.github/workflows/build-and-publish-image.yml

```diff
@@ -3,49 +3,72 @@ name: Build and publish plugin container image
 on:
   workflow_dispatch:
     inputs:
-      plugin:
-        description: 'The plugin name (e.g., flux)'
+      plugins:
+        description: 'Comma-separated list of plugin names to build (e.g., flux,cert-manager)'
         required: true
         type: string
       version:
         description: 'The plugin version (without a v prefix e.g., 0.1.0) - if not provided, will use version from package.json'
         required: false
         type: string
+      architectures:
+        description: 'Target architectures to build for'
+        required: true
+        type: choice
+        options:
+          - linux/amd64,linux/arm64
+          - linux/amd64
+          - linux/arm64
+        default: 'linux/amd64,linux/arm64'
 
 env:
   REGISTRY: ghcr.io
-  ORG: headlamp-k8s
-  PLUGIN: ${{ github.event.inputs.plugin }}
-  IMAGE_NAME: headlamp-plugin-${{ github.event.inputs.plugin }}
+  ORG: ${{ github.repository_owner }}
+  ARCHITECTURES: ${{ github.event.inputs.architectures }}
+  NODE_VERSION: '20'
```

Comment:
@joaquimrocha it's useful to change it in the GithubActions, but I'm not sure that the dependabot will work with this part.
Would you like, that I remove this changes in favor to use the Dockerfile definition?

Reply:
Sorry, what do you mean changing it in the GithubActions?
For me, the PR (untested) probably just needs the hashes back.

Reply:
Okay. Let's consider it later.